README.md

```diff
@@ -1,7 +1,53 @@
-# syzygy
+# Syzygy
+---
+## Recommended Setup Instructions
+0. Make sure you have `podman`, `python3`, `pip`, and `git` installed on your host machine
+1. Clone the syzygy repo and its submodules
+```
+git clone --recurse-submodules https://github.com/underground-software/syzygy.git
+```
+2. Clone the kdlp podman-compose repo
+```
+git clone https://github.com/underground-software/podman-compose.git
+```
+3. Set up a python virtual environment
+- Whenever you use `podman-compose.py`, make sure you use the `kdlp-venv` you set up! (run the `source` command listed below)
+```
+python3 -m venv kdlp-venv
```

Comment:
on fedora 39, you technically don't need to jump through all the hoops of making a venv and installing the dependencies. You can just install the distro provided podman-compose package which will install dependencies as global python packes and then then just clone the compose repo and run the .py file.